subsys/shell/shell.c

```diff
@@ -1163,6 +1163,13 @@ static void shell_log_process(const struct shell *shell)
 		processed = shell_log_backend_process(shell->log_backend);
 		shell_current_command_print(shell);
 
+		/* Arbitrary delay added to ensure that prompt is readable and
+		 * can be used to enter further commands.
+		 */
+		if (shell->ctx->cmd_buff_len) {
+			k_sleep(15);
```

Comment:
It would be more readable to do `k_sleep(K_MSEC(15))` here instead

Reply:
done